test/localize.js

```diff
@@ -22,7 +22,7 @@ describe( 'localize()', function() {
 
 		var LocalizedComponent = localize( MyComponent );
 
-		expect( LocalizedComponent.displayName ).to.equal( 'Localized' );
+		expect( LocalizedComponent.displayName ).to.equal( 'Localized' + MyComponent.name );
```

Comment:
Looks like this gives us a similar issue, MyComponent.name is an empty string :/

<img width="601" alt="screen shot 2016-10-19 at 22 10 29" src="https://cloud.githubusercontent.com/assets/4335450/19516696/e34b6996-9648-11e6-9a1a-23249bcf1d70.png">